app/src/main/java/org/pesmypetcare/mypetcare/services/PetManagerAdapter.java

```diff
@@ -21,42 +22,89 @@
+                PetManagerClient.BREED, pet.getBreed());
+            updateHealth(pet);
+        } catch (ExecutionException | InterruptedException e) {
+            e.printStackTrace();
+        }
+
+        /*ServiceLocator.getInstance().getPetManagerClient().updateGender(userToken, ownerUsername, name,
             pet.getGender().toString());
-        //ServiceLocator.getInstance().getPetManagerClient().updateBirthday(pet.getOwner().getToken(), ownerUsername,
-        //  name, pet.getBirthDate());
-        ServiceLocator.getInstance().getPetManagerClient().updateBreed(pet.getOwner().getToken(), ownerUsername,
+        ServiceLocator.getInstance().getPetManagerClient().updateBirthday(pet.getOwner().getToken(), ownerUsername,
+          name, pet.getBirthDate());
+        ServiceLocator.getInstance().getPetManagerClient().updateBreed(userToken, ownerUsername,
             name, pet.getBreed());
-        /*ServiceLocator.getInstance().getPetManagerClient().updatePathologies(pet.getOwner().getToken(),
-            ownerUsername, name, pet.getPathologies());*/
-        updateHealth(pet);
+        ServiceLocator.getInstance().getPetManagerClient().updatePathologies(pet.getOwner().getToken(),
+            ownerUsername, name, pet.getPathologies());
+        updateHealth(pet);*/
     }
 
     /**
      * Update the health data of the pet.
      * @param pet The pet to which its health data has to be updated
      */
-    private void updateHealth(Pet pet) {
+    private void updateHealth(Pet pet) throws ExecutionException, InterruptedException {
         String name = pet.getName();
         String ownerUsername = pet.getOwner().getUsername();
+        String userToken = pet.getOwner().getToken();
+
+        ServiceLocator.getInstance().getPetManagerClient().updateField(userToken, ownerUsername, name,
+            PetManagerClient.WEIGHT, pet.getWeight());
 
-        ServiceLocator.getInstance().getPetManagerClient().updateWeight(pet.getOwner().getToken(), ownerUsername,
+        /*ServiceLocator.getInstance().getPetManagerClient().updateWeight(pet.getOwner().getToken(), ownerUsername,
             name, pet.getWeight());
-        /*ServiceLocator.getInstance().getPetManagerClient().updateRecKcal(pet.getOwner().getToken(), ownerUsername,
+        ServiceLocator.getInstance().getPetManagerClient().updateRecKcal(pet.getOwner().getToken(), ownerUsername,
             name, pet.getRecommendedDailyKiloCalories());
         ServiceLocator.getInstance().getPetManagerClient().updateWashFreq(pet.getOwner().getToken(), ownerUsername,
             name, pet.getWashFrequency());*/
     }
 
     @Override
     public boolean registerNewPet(User user, Pet pet) {
-        ServiceLocator.getInstance().getPetManagerClient().createPet(user.getToken(), user.getUsername(), pet.getName(),
+        /*ServiceLocator.getInstance().getPetManagerClient().createPet(user.getToken(), user.getUsername(), pet.getName(),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 122).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5322199)